src/pages/OfferNew/Checkout/CheckoutContent.tsx

```diff
@@ -119,9 +119,9 @@ export const CheckoutContent: React.FC<Props> = ({
           onEmailChange={(email) => {
             const onCompletion = new Promise<void>((resolve, reject) => {
               Promise.all(
-                quoteIds.map((quoteId) => {
-                  editQuote({ variables: { input: { id: quoteId, email } } })
-                }),
+                quoteIds.map((quoteId) =>
+                  editQuote({ variables: { input: { id: quoteId, email } } }),
+                ),
```

Comment:
The battle of the tslints

Reply:
Actually, this is a bug fix :D since we didn't return the promise `editQuote` returns the `refetch` that should happen after happened at the same time as `editQuote` making members unable to sign unless they edit their ssn twice